yt/frontends/arepo/fields.py

```diff
@@ -125,12 +139,42 @@ def _el_number_density(field, data):
 
         if (ptype, "CosmicRaySpecificEnergy") in self.field_list:
 
+            self.alias(
+                (ptype, "specific_cosmic_ray_energy"),
```

Comment:
I am a bit confused reading the lines above - I assumed `ptype` was equal to `gas`. Did I miss something?

TBH the entire function looks very obscure to me and I am not used to dealing with particle-based datasets, so this may be me :)

Reply:
@cphyc in this case, `ptype` is `"PartType0"`, which is the gas type.